pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/osteo/OsteoAboutChildV2.java

```diff
@@ -43,10 +45,16 @@ public class OsteoAboutChildV2 implements CustomTask {
 
     private static final String STUDY_GUID = "CMI-OSTEO";
     private static final String ACTIVITY_CODE = "ABOUTCHILD";
+    private static final String UPDATES_DATA_FILE = "patches/about-you-child-updates.conf";
+    private static final String TRANS_UPDATE = "trans-update";
+    private static final String TRANS_UPDATE_OLD = "old_text";
+    private static final String TRANS_UPDATE_NEW = "new_text";
```

Comment:
I think that it would be better use UpdateTemplatesInPlace.traverseActivity()
You can find an example in this task - BrainConsentVersion2

Reply:
It's not about efficiency, it will make the code more readable, pretty and easier to maintain.